src/AutoRest.CSharp/Common/Input/CadlInputConstantConverter.cs

```diff
@@ -93,6 +93,22 @@ public static object ReadConstantValue(ref Utf8JsonReader reader, string propert
 
                     }
                     break;
+                case InputEnumType enumType:
+                    switch (enumType.EnumValueType.Kind)
+                    {
+                        case InputTypeKind.String:
+                            value = reader.GetString() ?? throw new JsonException();
```

Comment:
Please include the error message for easy trouble shooting.

Reply:
The occurs in json reader and we cannot know the detail, so we just return default exception.

Reply:
Can we just use the message like:  "The value is null for ......, expect 'string'."

Reply:
It may not because of the value is null or something. The message is also meaningless. Now we just use the default exception as we did before.